.vscode/settings.json

```diff
@@ -15,7 +15,9 @@
 	"jest.autoEnable": false,
 	"jest.pathToConfig": "./tests/js/jest.config.js",
 	"editor.defaultFormatter": "esbenp.prettier-vscode",
-	"editor.formatOnSave": true,
+	"[javascript]": {
```

Comment:
Hey @eugene-manuilov, was this change left in intentionally?

Reply:
Yes, it was. By default, if we use the `"editor.formatOnSave": true` setting, vscode formats everything, including scss files using the default prettier config which has conflict rules with our stylelint confing.

Reply:
Ok, thanks for clarifying! :)